live-examples/css-examples/images/function-repeating-linear-gradient.html

```diff
@@ -0,0 +1,29 @@
+<section id="example-choice-list" class="example-choice-list large">
+<div class="example-choice">
+<pre><code class="language-css">background: repeating-linear-gradient(#e66465, #e66465 20px, #9198e5 20px, #9198e5 25px);</code></pre>
+<button type="button" class="copy hidden" aria-hidden="true">
+    <span class="visually-hidden">Copy to Clipboard</span>
+</button>
+</div>
+
+<div class="example-choice">
+<pre><code class="language-css">background: repeating-linear-gradient(45deg, #3f87a6, #ebf8e1 15%, #f69d3c 20%);</code></pre>
+<button type="button" class="copy hidden" aria-hidden="true">
+    <span class="visually-hidden">Copy to Clipboard</span>
+</button>
+</div>
+
+<div class="example-choice">
+<pre><code class="language-css">background: repeating-linear-gradient(transparent, #4d9f0c 40px),
+           repeating-linear-gradient(0.25turn, transparent, #3f87a6 20px);</code></pre>
```

Comment:
super nit: this isn't quite aligned with the previous `repeating-linear-gradient` call.